Makefile

```diff
@@ -206,6 +216,12 @@ upstream.rebase:
 .pulumi/bin/pulumi: .pulumi/version
 	curl -fsSL https://get.pulumi.com | HOME=$(WORKING_DIR) sh -s -- --version $(cat .pulumi/version)
 
+$(PULUMICTL_BIN):
+	@mkdir -p $(BIN_DIR)
+	curl -L https://github.com/pulumi/pulumictl/releases/download/$(PULUMICTL_VERSION)/pulumictl-$(PULUMICTL_VERSION)-$(OS)-$(ARCH).tar.gz -o $(BIN_DIR)/pulumictl.tar.gz
+	tar -xzf .pulumi/bin/pulumictl.tar.gz -C $(BIN_DIR)
+	rm -f $(BIN_DIR)/pulumictl.tar.gz
+
 examples: install_equinix_plugin
 	scripts/generate_examples.sh
```

Comment:
I think the generate_examples script uses pulumictl too; if so it should be updated to use the copy in `.pulumi/bin`.